### DISCUSSION TITLE
Talk calls only available over LAN behind NGINX Proxy Manager

### DISCUSSION THREAD

**Discussion by DaleJP** [nextcloud/all-in-one#1040] [Questions] [answered]
I have just discovered that Talk isn't connecting audio or video calls behind NGINX Proxy Manager although I'm not sure that is the cause.

My setup:

Unifi router 
- 3478 TCP and UDP forwarded to NC server
- 80, 443 forwarded to NGINX
- local DNS record for cloud.mydomain.com to NC server

NC Server:
- Ubuntu Server
- Regular AIO script setup with guide adjustment for Reverse Proxy

NGINX Proxy Manager
- cloud.mydomain.com forwarded to NC server port 11000

What Works:
Everything on LAN
Text and notifications (including for calls) both WAN and LAN

What doesn't work:
Audio and video calls over WAN or between WAN and LAN

Over WAN, calls come through but when the second party tries to connect, it tries and then reverts to both ends "waiting for XX to join call" or sometimes seems to connect with no video or audio.

What I've tried:
- Removing the port forward rule on the router which seems to change nothing at all (Which I thought strange - I thought it might at least break something)
- Checked Talk settings in NC - Everything seems to be ok and reporting so.
- removing local DNS record for NC (no change)
- searching these discussions and there wasn't anything I could identify as being related.

Any help will be greatly appreciated.

**Comment by szaimen** [comment]
Sorry, cannot reproduce this. Is this only a problem with the talk android app?

**Reply by DaleJP** [reply]
Both iOS and Android. Haven't tried in a browser. 

It's really bizarre. 

Are there separate ports that Talk uses for audio and video? I could try opening those. 

I'm actually not sure where to check logs for calls to get a better idea of what's going on.

**Comment by szaimen** [comment, marked as answer]
I fear this is likely an issue of the android and probably ios apps as it works in a browser in my testing. See e.g. https://github.com/nextcloud/talk-android/issues/1712 for the android app.
